app/api/video/permissions.py

```diff
@@ -0,0 +1,22 @@
+import typing as tp
+
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+from rest_framework.request import Request
+
+from api.video.models import Video
+
+
+class IsOwnerOrReadOnlyVideoPermission(BasePermission):
```

Comment:
the code is the same as previous permission I guess. can you reduce duplication by inheritance?

Reply:
we cant, repeat only func has_permission, in the previous object was Account and this is the same as request.user in this case Account is object.account